test/e2e/artifact_test.go

```diff
@@ -0,0 +1,160 @@
+//go:build linux || freebsd
+
+package integration
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/containers/podman/v5/pkg/libartifact"
+	. "github.com/containers/podman/v5/test/utils"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gexec"
+)
+
+var _ = Describe("Podman artifact", func() {
+	BeforeEach(func() {
+		SkipIfRemote("artifacts are not supported on the remote client yet due to being in development still")
+	})
+
+	It("podman artifact ls", func() {
+		artifact1File, err := createArtifactFile(4192)
+		Expect(err).ToNot(HaveOccurred())
+		artifact1Name := "localhost/test/artifact1"
+		podmanTest.PodmanExitCleanly([]string{"artifact", "add", artifact1Name, artifact1File}...)
```

Comment:
For future reference,
```go
podmanTest.PodmanExitCleanly("artifact", "add", artifact1Name, artifact1File)
```
works as long as the number of parameters is a constant. `PodmanExitCleanly(array…)` remains an option if `array` is runtime-generated.